Dynamic-Programming/CoinChange.js

```diff
@@ -38,8 +81,24 @@ function minimumCoins (coins, amount) {
 function main () {
```

Comment:
This is not a must do change. If you are free, you can change this. I want you to remove the main function and instead implement a `jest` test file.

Reply:
Yes, you're right. It is not a must do change. I wanted to add the memoized approach for the above standard problem of Dynamic programming, so that readers could get both the approaches for solving the same.

Okay, I'll remove the `main` function and add a **jest spec file** instead.
Thanks